### DISCUSSION TITLE
Marble diagram is missing in the Operatos article in the docs

### DISCUSSION THREAD

**Discussion by filipkrw** [ReactiveX/rxjs#6454] [Report issues other than bug]
There's a dead image link in the "Marble diagrams" section of the [Operators article](https://rxjs.dev/guide/operators). It's kinda ironic, since the article says that "to explain how operators work, textual descriptions are often not enough" and "diagrams are often a better tool for that", but the diagram is missing :(

**Comment by raymonddavis** [comment]
The assest is not missing the src tag is incorrect.
Currently it is `../../src/assets/images/guide/marble-diagram-anatomy.svg`
it should be `../../assets/images/guide/marble-diagram-anatomy.svg`

There is an extra `src` in the path.

**Reply by filipkrw** [reply]
That's it, thanks for help.
